src/App.vue

```diff
@@ -153,9 +153,9 @@ export default {
     },
     // messages received from Stelace Dashboard
     receiveMessage (event) {
-      const origins = this.allowedMessageOrigins.split(',')
+      const origins = this.allowedMessageOrigins.split(',').map(o => o.trim())
 
-      if (this.allowedMessageOrigins !== '*' && !origins.includes(event.origin)) return
+      if (!origins.includes('*') && !origins.includes(event.origin)) return
```

Comment:
This is not equivalent to my suggestion. You're not supporting wildcard such as `*.example.com` here but making it always pass which is unsafe.